guardian/pkg/config/tigera.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/kelseyhightower/envconfig"
+
+	"github.com/projectcalico/calico/guardian/pkg/server"
+)
+
+func NewTigeraConfig() (*TigeraConfig, error) {
+	cfg := &TigeraConfig{}
+	if err := envconfig.Process(EnvConfigPrefix, cfg); err != nil {
+		return nil, err
+	}
+
+	cfg.configureLogging()
+
+	return cfg, nil
+}
+
+type TigeraConfig struct {
+	CalicoConfig
+
+	PacketCaptureCABundlePath string `default:"/certs/packetcapture/tls.crt" split_words:"true"`
```

Comment:
Are these needed by this component ?

Reply:
No I didn't actually mean to commit this, I'll remove this (it's for private).